ChangeLog.d/9302.txt

```diff
@@ -0,0 +1,8 @@
+Features
+   * When the new compilation option MBEDTLS_PSA_STATIC_KEY_SLOTS is enabled,
+     key store uses static key slot buffers instead of dynamically allocated
+     ones.
+   * The size of each static key slot buffer is determined by
+     MBEDTLS_PSA_STATIC_KEY_SLOT_BUFFER_SIZE. If not set, its default value
+     is set to accomodate for the largest PSA asymmetric/symmetric key enabled
+     in the build.
```

Comment:
I don't think `MBEDTLS_PSA_STATIC_KEY_SLOT_BUFFER_SIZE` is really a separate feature from `MBEDTLS_PSA_STATIC_KEY_SLOTS`. Rather, I think it should be mentioned in the entry for `MBEDTLS_PSA_STATIC_KEY_SLOTS`. E.g.

> (…) The buffer size is given by the ption MBEDTLS_PSA_STATIC_KEY_SLOT_BUFFER_SIZE. By default, it accommodates the largest expected key.

Reply:
This reads like it's part of the previous bullet point, but fell out. If we want it as a separate bullet point, it should start something like "Add new config option MBEDTLS_PSA_STATIC_KEY_SLOT_BUFFER_SIZE, which ..."